benchmark/testFiles/config/ConfigLoader.ts

```
@@ -0,0 +1,23 @@
+import fs from 'fs';
+
+export class ConfigLoader {
+    private static config: any = null;
+    
+    static async loadConfig(env: string = 'development') {
+        const configFile = `config.${env}.json`;
+        
+        if (!this.config) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)